ci/templates/steps.yml

```diff
@@ -17,7 +17,7 @@ steps:
       mavenPomFile: "pom.xml"
       mavenOptions: "$(MAVEN_OPTS)"
       options:
-        "compile -DskipTests -P build-web-console  -Dhttp.keepAlive=false"
+        "compile  javadoc:javadoc -DskipTests -P build-web-console,maven-central-release  -Dhttp.keepAlive=false"
```

Comment:
it is not a good idea to run `maven-central-release` on CI, it does more than javadoc. Perhaps we can have a new profile just for javadoc ? or force javadoc without using  `maven-central-release` profile?

Reply:
As far as I can see `maven-central-release` only does `javadoc` if the goal is `javadoc:javadoc`. Other actions are on `deploy`

Reply:
I added new profile